include/deal.II/grid/tria_faces.h

```diff
@@ -70,14 +70,29 @@ namespace internal
        *
        * @note Used only for dim=3.
        */
-      std::vector<unsigned char> quads_line_orientations;
+      std::vector<bool> quads_line_orientations;
```

Comment:
Can you update the comment to say true means flipped (I assume this is what it is)?

Reply:
Yes. I'm also writing more general documentation for how we handle orientations since its a tricky area.

Reply:
FYI: unlike 'flip' and 'rotate', for 'orientation' `true` is the default and `false` is the reverse.